imitation/input_fn.py

```diff
@@ -218,5 +218,6 @@ def evaluation_input_fn(tfrecord_fpaths, batch_size):
         mode=tf.estimator.ModeKeys.EVAL,
         num_epochs=1,
         model_preprocessors=[FilterValidIntention(), CarlaPreprocessor()],
+        num_parallel_calls=16,
```

Comment:
Should this not be added also to the train input function? Or are you skipping that due to the instability issues you observed?

Reply:
I skipped this for instability reasons for now, also given that there is a `num_parallel_calls` on each `ProbabilisticImageAugmentor` already.